pages/jobs/JobsFilter.jsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Heading,
+  Stack,
+  Text,
+  Wrap,
+  WrapItem,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { MdFilterList } from "react-icons/md";
+
+import { FilterTagButton } from "@/components/FilterTagButton";
+import { jobLocationsArray } from "@/utils/helpers/jobLocationsArray";
+import { jobTagsArray } from "@/utils/helpers/jobTagsArray";
+
+export const JobsFilter = ({
+  locationFiltersList,
+  tagFiltersList,
+  handleTagSelect,
+  handleLocationSelect,
+  handleClickRemoteOnly,
+}) => {
+  const [showMoreTags, setShowMoreTags] = useState(false);
+
+  const jobTagsByCategory = jobTagsArray.reduce((acc, curr) => {
+    if (!acc[curr.category]) {
+      acc[curr.category] = [];
+    }
+    acc[curr.category].push(curr);
+    return acc;
+  }, {});
+
+  return (
+    <Stack padding="8" borderWidth="1px" borderRadius="20px" spacing={4}>
+      <Box>
+        <Heading as="h5" marginBottom="4" size="md">
+          Narrow your search
+        </Heading>
+        <Button
```

Comment:
I'm thinking we need some kind of active state for this (which I totally didn't convey in the design) - or at the very least changing the text to "Showing Remote Only" or something like that. Just a quick idea - using green 600 for a bit more contrast and since we're using green for the other filters. 

Thoughts? 

<img width="258" alt="Screen Shot 2022-06-09 at 3 28 53 PM" src="https://user-images.githubusercontent.com/1581290/172948526-2d8d2eec-f9f9-4120-a406-e8eff6b54d53.png">

Reply:
I agree. Added this as `colorScheme: 'green'`